assets/javascript/app.js

```diff
@@ -0,0 +1,93 @@
+// namespace object for food-finding functions
+var rest_test = {
+  
+  restaurants: [{
+    name: "McDonald's",
+    image:"url",
+    foodtype:"American",
+    description:"fast food",
+    link:"url"
+  }],
+
+  getNearbyRestaurants: function() {
+    return restaurants[Math.floor(Math.random()*restaurants.length)];
+  },
+
+  setFoodClick: function() {
+    $("#findfood").click(function() {
```

Comment:
Does putting an on-click button within another function work? I feel like this should be outside of the object definition.